tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/rollback-after-failed-upgrade-test.go

```diff
@@ -62,6 +53,22 @@ var _ = Describe(
 				_, err := newIbguBuilder.DeleteAndWait(1 * time.Minute)
 				Expect(err).ToNot(HaveOccurred(), "Failed to delete IBGU on target hub cluster")
 
+				abortIbguBuilder := ibgu.NewIbguBuilder(TargetHubAPIClient, "abortibgu", tsparams.IbguNamespace)
```

Comment:
To stay consistent with the other IBGU builder definitions, should we use method chaining like this:

abortIbguBuilder := ibgu.NewIbguBuilder(TargetHubAPIClient, "abortibgu", tsparams.IbguNamespace).
	WithClusterLabelSelectors(tsparams.ClusterLabelSelector).
	WithSeedImageRef(CNFConfig.IbguSeedImage, CNFConfig.IbguSeedImageVersion).
	WithPlan([]string{"Abort"}, 5, 10)

Reply:
sure, updated. thanks!